llvm/test/Transforms/OpenMP/deduplication_soundness.ll

```diff
@@ -0,0 +1,70 @@
+; RUN: opt -passes=openmp-opt-cgscc -S < %s | FileCheck %s
```

Comment:
For a test like this I'd recommend generating the check lines with `llvm/utils/update_test_checks.py --function-signature`.

Reply:
Thanks! I'd prefer this too, if possible; will see if there's a way to avoid generating more check lines than I already have (in the past had to use `UTC_ARGS` for in certain cases, but perhaps won't be needed here).

Reply:
Done!